scripts/windows/build-dotnet-trace.bat

```diff
@@ -0,0 +1,15 @@
+IF NOT EXIST .\dep\dotnet-install.ps1 (
```

Comment:
Just a suggestion - rename to `build_dotnet_trace.bat`, as we use mostly underscores for script names.